### DISCUSSION TITLE
Interpreter(): "JavascriptError: TypeError: obj is null" when running print() with end=""

### DISCUSSION THREAD

**Discussion by findersjeepers** [brython-dev/brython#2385] [Q&A]
I was surprised that of all the things that could have possibly gone awry with my attempts to experiment with the Interpreter, it was a print() command that caused an issue, at least according to the error in the console. Apparently it does not like `end=""`. As a Not Programmer, I don't know why this is the case; might anyone here know the reason, or how I ought to get around it?

**Comment by PierreQuentel** [comment]
Thanks for reporting this, it's a bug and I fixed it in commit 84fb47b5fe2836a98d43c1e7eec4ac842ce50951
